MekHQ/src/mekhq/gui/model/UnitTableModel.java

```diff
@@ -352,19 +305,16 @@ public Component getTableCellRendererComponent(JTable table, Object value, boole
                 } else if (u.getActiveCrew().size() < u.getFullCrewSize()) {
                     applyColors(colors.getUncrewed());
                 } else {
-                    setBackground(UIManager.getColor("Table.background"));
                     setForeground(UIManager.getColor("Table.foreground"));
+                    setBackground(UIManager.getColor("Table.background"));
```

Comment:
Why does this order matter and others do not seem to?

Reply:
It doesn't, I removed it as part of my testing and put it back in a different order... and didn't consider it worth a commit to swap back.